<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Приложение шифрования</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выберите тип шифруемого файла</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_Word">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_jpg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>jpg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_bmp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>bmp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_txt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>txt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_wav">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>wav</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Зашифровать файл</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>push_Word</zorder>
  <zorder>push_jpg</zorder>
  <zorder>push_bmp</zorder>
  <zorder>push_txt</zorder>
  <zorder>push_wav</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
